public/app/features/scenes/dashboard/PanelTimeRange.ts

```diff
@@ -0,0 +1,66 @@
+import { dateMath, getTimeZone, TimeRange, TimeZone } from '@grafana/data';
+
+import { SceneObjectBase } from '../core/SceneObjectBase';
+import { SceneTimeRangeLike, SceneTimeRangeState } from '../core/types';
+
+export interface PanelTimeRangeState extends SceneTimeRangeState {
+  timeShift?: string;
+}
+
+export class PanelTimeRange extends SceneObjectBase<PanelTimeRangeState> implements SceneTimeRangeLike {
```

Comment:
I like this idea! But I think it would require more changes than just the localized time range. I.e. given the Query runner is attached to the root of the scene, we have to add a mechanism that will identify which panel requested a query with overridden range, so that other panels that consume the data via `sceneGraph.getData()` won't get updates.

Reply:
@dprokop not sure I understand your comment. Query runners for dashboard panels are not attached to the roof of the scene but to each VizPanel. 

> we have to add a mechanism that will identify which panel requested a query with overridden range, 

?? the query runner will get the closest scoped time range. Confused

Reply:
I'm thinking more of future use cases. I.e. Two panels sharing the same query runner when comparing data from different ranges. Would probably be nice to have a single source of queries and only the execution context changes. But - we may leave that problem for now since it will work with multiple SceneQueryRunners.